app/assets/javascripts/channels/concurrent_editing.js

```diff
@@ -15,6 +15,10 @@ App.concurrent_editing = App.cable.subscriptions.create("ConcurrentEditingChanne
   },
 
   speak: function(changes) {
+    /* Called when an image is updated from Map.js ('saveImage' function).
+    *  This function calls concurrent_editing_channel.rb's 'sync' method
```

Comment:
Please don't write `This`. 
Reason a new developer writes comments between 18 and 19 LOC then this will become outdated and ambiguous.